app/router.coffee

```diff
@@ -5,6 +5,8 @@ Router = Ember.Router.extend
   location: ENV.locationType
   rootURL: ENV.rootURL
 
+Ember.$('body').addClass('theme-' + ENV.whitelabel.theme)
```

Comment:
@yashwin Let me know if there is a better way of setting theme class ^

Reply:
@Praseetha-KR : you can do that 👍